config/deploy/production.rb

```diff
@@ -1,34 +1,3 @@
-#
-# For a detailed explanation of roles (ie :web, :app, etc) see:
-# http://stackoverflow.com/questions/1155218/what-exactly-is-a-role-in-capistrano
-#
-# The :primary attribute is used for tasks we only want to run on one machine
-# 
-require 'capistrano/gitflow_version'
```

Comment:
Why are we loosing this ?

Reply:
We're not, I moved the comments to the top, the `require` is on line 10.